owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileActivity.java

```diff
@@ -316,12 +317,22 @@ public void onRemoteOperationFinish(RemoteOperation operation, RemoteOperationRe
         }
     }
 
-    protected void showRequestAccountChangeNotice() {
-        Snackbar.make(findViewById(android.R.id.content), R.string.auth_failure_snackbar, Snackbar.LENGTH_INDEFINITE)
-                .setAction(R.string.auth_failure_snackbar_action, v ->
-                        startActivity(
-                                new Intent(FileActivity.this, ManageAccountsActivity.class)))
-                .show();
+    protected void showRequestAccountChangeNotice(String errorMessage, boolean mustChange) {
+        if (mustChange) {
+            new AlertDialog.Builder(this)
```

Comment:
Not sure how difficult would be to create an Activity kotlin extension to show alert dialogs. There's some `showAlertDialog` calls in the project code that could make use of it, apart from being used here.

Reply:
Yes, it good be good to unify them, but i think that it could be done out of this pr.